compact/hcal/forward_insert.xml

```diff
@@ -84,27 +83,6 @@
       <backplate thickness="HcalEndcapPInsertBackplateThickness"/>
       <comment> Front layer to match front walls of LFHCAL modules </comment>
       <comment> Slices will be ordered according to the slice order listed here </comment>
-      <layer repeat="1" thickness="HcalEndcapPInsertFrontLayerThickness" vis="InvisibleWithDaughters">
-        <slice material="Steel235" thickness="LFHCAL_FrontWallThickness" vis="LFHCAL8MModVis" />
```

Comment:
Front wall is gone? Then, let's remove the constant definition for its thicknesses `HcalEndcapPInsertFrontLayerThickness` and `LFHCAL_FrontWallThickness`.

Reply:
I mistakenly thought this was just an ordinary layer of the absorber and added it to the count of other absorbers, however apparently this has a somewhat different thickness.  I'll change this so that it is frontplate layer + 64 "normal" layers.

Reply:
the front wall has been reintroduced in the most recent commit.